content/en/docs/tasks/debug/debug-cluster/_index.md

```diff
@@ -14,6 +14,12 @@ problem you are experiencing. See
 the [application troubleshooting guide](/docs/tasks/debug/debug-application/) for tips on application debugging.
 You may also visit the [troubleshooting overview document](/docs/tasks/debug/) for more information.
 
+{{% alert title="Note" %}}
```

Comment:
This is a note, please don't make it an alert.
Actually, you may want to remove this shortcode.